docs/03-guides/01-repositories/02-the-platform.md

```diff
@@ -1,74 +1,89 @@
-| `helm/`                      | Helm charts for deploying the Teacher Portal and related services to Kubernetes.                |
-| `prototypes/`                | A previous team's UI prototypes for the Teacher Portal.                                         |
-| `resources/`                 | A previous team's sprint reports and related documentation.                                     |
+| `helm/` (obsolete)           | Helm charts for deploying the Teacher Portal and related services to Kubernetes.                |
+| `scripts/`                    | Helper scripts for CI/CD.                                                                      |
 | `server/`                    | Teacher Portal's backend code.                                                                  |
-| `.gitignore`                 | Specifies files that are ignored and untracked by Git.                                          |
-| `CHANGELOG.md`               | A previous teams's commit history. Contains information on feature additions and bug fixes.     |
 | `docker-compose.yml`         | Docker Compose configuration for running the Teacher Portal locally in containers.              |
 | `openapi.yml`                | Teacher Portal's API specifications. Uses the OpenAPI specification standard.                   |
-| `README.md`                  | Documentation for setting up and running the Teacher Portal locally or on a server. A lot of this documentation is now outdated. |
 
----
+> Helm is obsolete now as the live deployment was drastically simplified in the migration from AWS to GCP in 2025. Kubernetes was removed from the project in favour of a simple VM setup.
 
-## Running the Teacher Portal Locally
+### Frontend / `client`
 
-### Prerequisites
+The frontend of the Teacher's Portal is made with React v17.0.2 (October 2022). React has seen at least two major versions since then, but updating the project's version would be a significant task given the number of breaking changes and incompatible dependencies that would be introduced. 
 
-- [Docker](https://www.docker.com/products/docker-desktop)
+There are tests set up for the React project that don't appear to have been utilized since the initial development work in 2021, as no subsequent team has completed any significant refactors of the frontend. If your team is completing work on updating, fixing, or completing the Teacher's Portal, these will probably be important to investigate.
 
-### Instructions
+The `AutHandler` defined in `App.tsx` is where the current link between Keycloak validation and the user's in the existing database is made. Because the database uses integers for the user ids and Keycloak uses uuids, the users are instead signed-in via their email address here, as this is consistent between both. The password validation is handled by Keycloak, and has therefore been disabled in the Teacher's Portal frontend.
```

Comment:
```suggestion
The `AuthHandler` defined in `App.tsx` is where the current link between Keycloak validation and the user's in the existing database is made. Because the database uses integers for the user ids and Keycloak uses uuids, the users are instead signed-in via their email address here, as this is consistent between both. The password validation is handled by Keycloak, and has therefore been disabled in the Teacher's Portal frontend.
```